.github/workflows/playwright.yml

```diff
@@ -0,0 +1,78 @@
+name: Playwright E2E Tests
+
+on:
+  push:
+    branches:
+      - "develop"
+      # - "feature/**"
+  pull_request:
+    types: [opened, synchronize, reopened]
+  # Allows workflow to be called from other workflows
+  workflow_call:
+    inputs:
+      ref:
+        required: true
+        type: string
+
+# Avoid duplicate workflows on same branch
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-playwright
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest-8-cores
```

Comment:
should we increase this to `ubuntu-latest-16-cores`? I'm not sure if that costs more money
cc: @kmcgrady @mayagbarnes

Reply:
We don't need a better machine for now since the total time required for the Playwright CI workflow is still way under the total time for other CI workflows. But once we migrate more tests, this would be a very easy step to speed up playwright since everything is already completely parallelized based on the available CPUs.

Reply:
When I did GHA runner analysis for the Cypress workflow it showed no meaningful improvement from `8-core` to `16-core` runners so I agree we should at most be using the `8-core` now. But since we have a limited number of upgraded `8-core` runners, I would be curious if the run time comparison from default `4-core` is meaningful - could you test?
I'm thinking perhaps we use the `4-core` for now, and then transition to `8-core` when we move from Cypress - that way we don't slow development down waiting for runners in the meantime.

Reply:
Since the playwright tests are nearly fully parallelizable, every additional core will speed up the test execution. But I think we could go to `4-core` temporarily since the workflow is still much faster than our other slowest workflows.

Reply:
Downgrading to `4-core` makes the workflow run for 15 minutes, which is good for now 👍